.github/workflows/spread-tests.yaml

```
@@ -13,6 +13,10 @@ on:
         description: 'The spread backend to use (for possible values, check spread.yaml). This cannot be empty'
         required: true
         type: string
+      alternative-backend:
+        description: 'The spread backend to use when the backend cannot be used'
+        required: true
```

Since not all groups have alternative backends, I would make this not required
```suggestion
        required: false
```

---

right, Initially I set it as false, not sure when it became true